logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/conversation/message/ApplicationMessageHandler.kt

```diff
@@ -120,9 +120,11 @@ internal class ApplicationMessageHandlerImpl(
                     status = Message.Status.SENT,
                     editStatus = Message.EditStatus.NotEdited,
                     visibility = visibility,
-                    expectsReadConfirmation = content.expectsReadConfirmation
+                    expectsReadConfirmation = content.expectsReadConfirmation,
+                    expirationStatus = content.expiresAfterMillis?.let { Message.ExpirationStatus.Expiratable(it) }
```

Comment:
`Message.ExpirationStatus.Expiratable` where is this defined? I can't find it when searching this PR or the branch?

Reply:
oops I forgot about that part